build/package/binaries/linux/installer.sh

```diff
@@ -255,6 +255,9 @@ install_agent() {
             echo "Adding kernel capabilities..."
             eval "${set_cap} 'CAP_SYS_PTRACE,CAP_DAC_READ_SEARCH=+ep' ${binary}" || exit 1
         fi
+
+        chown "${NRIA_USER}" "${binary}"
+        chmod 0754 "${binary}"
```

Comment:
who was the owner/group before this change? if it was root and had 0755 rights, what do you think about doing:
```
chown ":${NRIA_USER}" "${binary}"
chod 0750 "${binary}"
```
?

Reply:
Hmm, seeing what [L247](https://github.com/newrelic/infrastructure-agent/pull/1592/files/7b7a5addabb6afb77812e86f8730fc504098ed29#diff-84ba09b15a3e7dabc395d96eebed28f7229979df5dd33db15782d6fe4ef9e42bR247) was doing I figured `NRIA_USER` had to be owner of the binary as well. If that's not actually needed then this would be a better approach.

However, I see that there is no mention of a custom group in `installer.sh`, only user (see for example [L148](https://github.com/newrelic/infrastructure-agent/pull/1592/files/7b7a5addabb6afb77812e86f8730fc504098ed29#diff-84ba09b15a3e7dabc395d96eebed28f7229979df5dd33db15782d6fe4ef9e42bR148)). What if the group `NRIA_USER` does not exist? Perhaps it's created elsewhere before this is run?